python/uagents-core/uagents_core/contrib/protocols/chat/__init__.py

```diff
@@ -122,3 +127,61 @@ class ChatAcknowledgement(Model):
         ChatAcknowledgement: set(),
     },
 )
+
+
+# --- Helper Class for simplifying Chat Protocol -----------------------------
+
+
+class ChatObjects:
+    """
+    Utility class for constructing ChatMessage objects with common content types.
+    """
```

Comment:
It's not clear to me what the value of this class is over standalone utility functions. Maybe it would help if you add an example usage somewhere, maybe in a README?

I have a slight preference for preserving something close to the interface for `create_text_chat` that is used in a lot of agents already (e.g. https://github.com/fetchai/uAgent-Examples/blob/main/6-deployed-agents/geo/google-api-geolocation-agent/chat_proto.py#L22-L30). The reason is that this would make refactoring these agents a bit simpler. But I'd be open to other suggestions if you have a compelling reason or example.